src/utils/helpers.ts

```diff
@@ -207,8 +208,11 @@ export function getTime() {
 }
 
 export function getCurrentLanguage(guildID: string) {
-  return botCache.guildLanguages.get(guildID) ||
-    cache.guilds.get(guildID)?.preferredLocale || "en_US";
+  return (
```

Comment:
Whats this new formatting?

Reply:
Sorry, it's my vscode settings. I'll see if I can set it to the same as the project uses.

Reply:
Ok, I spent an hour trying to figure this out but it seems to be part of prettier defaults.

Reply:
Yup, it's definitely prettier defaults. I guess that should be fine? 
```
["INFO" - 8:38:19 PM] No local configuration (i.e. .prettierrc or .editorconfig) detected, falling back to VS Code configuration
["INFO" - 8:38:19 PM] Prettier Options:
{
  "arrowParens": "always",
  "bracketSpacing": true,
  "endOfLine": "lf",
  "htmlWhitespaceSensitivity": "css",
  "insertPragma": false,
  "jsxBracketSameLine": false,
  "jsxSingleQuote": false,
  "printWidth": 80,
  "proseWrap": "preserve",
  "quoteProps": "as-needed",
  "requirePragma": false,
  "semi": true,
  "singleQuote": false,
  "tabWidth": 2,
  "trailingComma": "es5",
  "useTabs": false,
  "vueIndentScriptAndStyle": false,
  "filepath": "/home/emanuel/source/boilerplate/src/utils/helpers.ts",
  "parser": "typescript"
}
```

Reply:
We don't use prettier but denos default formatter.